NSS-Day-Cohort-42/holiday-road-eccentric-eels
# Refactored itin list to show saved itineraries on page load. Refined weather widget.

# Steps to Review

1. `git fetch --all`
1. `git checkout mp-add-destination-images`
1. make sure you are serving `json-server -w itineraries.json -p 8087` from the API folder
1. make sure you are serving `json-server -w parkData.json -p 8088` from the API folder
1. make sure you are serving to your browser
1. review code to ensure logic is sound
1. review in browser to verify that the itineraries populate on page load, weather widget populates after park selected, and all components react to window resize.
